test/parallel/test-cluster-inspect-port-manual.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+/**
+ * This test suite checks debug port numbers
+ * when node started without --inspect argument
+ * and debug initiated with cluster.settings.execArgv or similar
+ */
+
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+
+if (cluster.isMaster) {
+  let message = 'If master is not debugging, ' +
+                'forked worker should not have --inspect argument';
+
+  cluster.fork({message}).on('exit', common.mustCall(checkExitCode));
+
+  message = 'When debugging port is set at cluster.settings, ' +
+            'forked worker should have debugPort without offset';
+  cluster.settings.execArgv = ['--inspect=' + common.PORT];
```

Comment:
@mutantcornholio Yeah, you're right, but it didn't quite work out that way because other tests in parallel use port `0` (to get the operating system to assign an available port) and those tests will sometimes end up occupying `common.PORT`. Not a super-common cause of test failures, but it has been seen in CI.